File: 268. Missing Number/268. Missing Number.swift
class Solution {
    func missingNumber(_ nums: [Int]) -> Int {
        var num = nums
        num.sort()
        var i = 0
        while i < num.count{ 
            if num[i] != i {
                return i
            }
            else {
                i += 1
            }
        }
        return nums.count
    }
}

File: 268. Missing Number/268. Missing Number.py
class Solution(object):
    def missingNumber(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        i = 0
        lenn = len(nums)
        nums.sort()
        for i in range(lenn):
            if nums[i] != i:
                return i
        return i+1
